modules/percolator/src/test/resources/rest-api-spec/test/mpercolate/10_basic.yaml

```diff
@@ -1,56 +0,0 @@
----
-"Basic multi-percolate":
```

Comment:
Agreed, I added a basic test for both the search and msearch apis using a `percolate` query.